dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/utils/AcmeFatUtils.java

```diff
@@ -276,6 +276,30 @@ public static void configureAcmeCA(LibertyServer server, CAContainer caContainer
 	 */
 	public static void configureAcmeCA(LibertyServer server, CAContainer caContainer,
 			ServerConfiguration originalConfig, boolean useAcmeURIs, boolean disableRenewWindow, String... domains) throws Exception {
+		configureAcmeCA(server, caContainer, originalConfig, useAcmeURIs, disableRenewWindow, false, domains);
```

Comment:
I'm afraid that by defaulting this to false, everytime we start a server we are going to refresh the certificate. This may result in test passing not for the reason the test was written. I think this should default to "true" to disable it by default. And only the tests that are testing history should enable it. I know this is backwards from how we would run, but i think it fits best with our existing tests.

Reply:
As discussed, I switched the testing default to True.